cmd/send-email/main.go

```diff
@@ -20,11 +22,23 @@ func main() {
         kong.Description("Send an email."),
     );
 
+    db.MustOpen();
```

Comment:
The DB needs to be opened after the args are fully checked.
This is because the arguments/config can set where the DB lives.

Note that the previous comment mentioned the correct ordering:
> The DB should generally be opened as the first thing after checking the args,
> since it is used for things like logging.
> So even in the case where no course is provided, the DB is required.